### PR TITLE
Fix a couple small issues

### LINKED ISSUE #1934: API Consistency: Branches vs BranchInfos
`ListCommit()` returns a `CommitInfos` message which has a `commit_info` field
`ListRepo()` returns `RepoInfos` message which has a `repo_info` field
`ListFile()` returns `FileInfos` message which has a `file_info` field
`ListJob()` returns `JobInfos` message which has a `job_info` field
`ListPipeline()` returns `PipelineInfos` message which has a `pipeline_info` field

...but for some reason `ListBranch()` returns a `Branches` message which has a `branches` field

### PR DESCRIPTION
**Pull request by derekchiang**
* Rename `JOB_STOPPED` to `JOB_KILLED`
* Rename `PIPELINE_STOPPED` to `PIPELINE_PAUSED`
* Rename `pfs.Branch` to `pfs.BranchInfo` and `pfs.Branches` to `pfs.BranchInfos`.
* Make it clear that to set description for a pipeline, use the pipeline spec.

Fix #1934, #1921, #1926

### CODE REVIEW

File: src/server/vendor/migration/onefoureight/db/pps/pps.pb.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ import google_protobuf "github.com/gogo/protobuf/types"
 import google_protobuf1 "github.com/gogo/protobuf/types"
 import google_protobuf2 "github.com/gogo/protobuf/types"
 import _ "github.com/gogo/protobuf/gogoproto"
-import pfs "github.com/pachyderm/pachyderm/src/client/pfs"
```
Comment (jdoliner) [inline src/server/vendor/migration/onefoureight/db/pps/pps.pb.go]: Was this done by manually editing the `pps.proto` file? If so won't this change disappear if you do `make proto`

Comment (derekchiang) [inline src/server/vendor/migration/onefoureight/db/pps/pps.pb.go]: This proto file is manually copied from 1.4.8.  `make proto` doesn't affect it.

Comment (jdoliner) [inline src/server/vendor/migration/onefoureight/db/pps/pps.pb.go]: I see, makes sense.

**Comment by gabrielgrant** [comment]
Thanks for getting around to these, @derekchiang 

The one other description-related issue I'd mentioned (no way to edit repo descriptions) didn't seem to have an issue, so I opened one: #1995

**Pull request merged**